package.json

```diff
@@ -29,8 +29,7 @@
     "@redocly/cli": "^1.20.0",
     "chalk": "^5.3.0",
     "classnames": "^2.3.2",
-    "convex": "^1.14.1",
-    "convex-helpers": "file:packages/convex-helpers/dist",
+    "convex": "^1.17.0",
```

Comment:
unless @ianmacartney wants this it seems unintentional

Reply:
yeah - the helpers import shouldn't be deleted - and you can test this yourself by running the example page from the root to verify that things work as expected.